cluster-service/Makefile

```diff
@@ -19,14 +19,19 @@ deploy:
 			-g ${RESOURCEGROUP} \
 			-n clusters-service \
 			--query clientId) && \
+	AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID=$(shell az ad app list \
+			--display-name aro-hcp-dev-first-party \
+			--query '[*]'.appId \
+			-o tsv) && \ 
```

Comment:
@geoberle @janboll can you confirm if this is the correct command to get the client id of the fpa identity? The returned value is currently this: `fc4c229d-7c1c-4f84-a4fc-eef14d762327`

Reply:
yes i think so. for a regular managed identity, it's client ID matches the appId of the backing app

Reply:
thanks!